x-pack/metricbeat/module/sql/query/_meta/data_mysql_variables_old.json

```diff
@@ -0,0 +1,509 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "event": {
+        "dataset": "sql.query",
+        "duration": 115000,
+        "module": "sql"
+    },
+    "metricset": {
+        "name": "query",
+        "period": 10000
+    },
+    "service": {
+        "address": "localhost:54399",
+        "type": "sql"
+    },
+    "sql": {
+        "driver": "mysql",
+        "metrics": {
+            "numeric": {
+                "aborted_clients": 0,
```

Comment:
Smaller example here too, please.

Reply:
This file is only used by integration test for now and not in the external doc, so leaving as-is.